deeplake/api/dataset.py

```diff
@@ -345,12 +346,12 @@ def empty(
         """Creates an empty dataset
 
         Args:
-            path (str, pathlib.Path): - The full path to the dataset. Can be:
-                - a Deep Lake cloud path of the form ``hub://username/datasetname``. To write to Deep Lake cloud datasets, ensure that you are logged in to Deep Lake (use 'activeloop login' from command line)
+            path (str, pathlib.Path): - The full path for storing to the Deep Lake dataset. It can be:
```

Comment:
`The full path to the dataset.` sounds correct.

Reply:
to be hones I don't see any particular difference. Can proceed with suggestion